test_calculus.py

```diff
@@ -1,9 +1,15 @@
 """
 test_calculus.py
 """
-import calculus as calc
+import pytest
+from calculus import bisection_scipy
```

Comment:
delete this one

Reply:
I changed the import statements and tried to sync the repo for linting however the pylint review has been stuck on 
"Expected — Waiting for status to be reported" for a while, what course of action can I take.